docker-compose.yml

```diff
@@ -8,5 +8,6 @@ services:
       PAGES_CKAN_URL: $PAGES_CKAN_URL
       NEXT_PUBLIC_TRACKING_ID: $TRACKING_ID
       NEXT_PUBLIC_TAG_ID: $GOOGLE_TAG_ID
+      NEXT_PUBLIC_ENV: $ENV
```

Comment:
Not 100% sure this is how the `environment` name is passed around in github workflows, but we can deploy and find out...